### ISSUE TITLE
Missing HTTP Status Code for response in Kafka message (Req_URL)

### ISSUE THREAD

**Issue by davidFR** [buger/goreplay#868]
With version master, Req_URL for response (HTTP Status Code) are missing in Kafka message.

Use case : record an application without Reason Phrase in HTTP Status
```
HTTP/1.1 200 
Content-Type: application/json; charset=utf-8
[...]
```

Recorded with --output-kafka-host xxx : 
```
{"Req_URL":"","Req_Type":"2","Req_ID":"fd43e7ee164f12e0f5c2ad3d", ..... }
```

Reason: 
Function proto.Status use Proto.Path, but Proto.Path was changed in https://github.com/buger/goreplay/commit/fdc8b094f060fc8afc379d8cfd105426b6e12f30#diff-56e029c4171b6506a9438d83466162baa2cd99616463e670b6186bcf7eafa6e0L295

Now Proto.Path can't manage HTTP Status without Reason Phrase, allowed by spec (https://www.w3.org/Protocols/rfc2616/rfc2616-sec6.html)

Note that HTTTP2 does not define a way to carry reason phrase, so it is important to comme back to the previous.

**Comment by davidFR** [comment]
Two solutions : 
* come back to the previous code (but it is correct for Path)
* or change code only for Status (I choose) and modify Kafka_ouput to deal with Path and Status accordng to Req_Type (I choose this option in my PR)

**Comment by urbanishimwe** [comment]
@davidFR we should add another field `Status(Req_Status)` that should be filled for response status and left empty for requests.

> Now Proto.Path can't manage HTTP Status without Reason Phrase, optional in HTTP1.1 spec

I think the spec says _"The client is not required to examine or display the Reason- Phrase."_ but it **does not** mark it as optional.

**Comment by davidFR** [comment]
We have Spring Boot Applications like zuul gateway (https://github.com/Netflix/zuul) that doesn't provide Reason Phrase, so we have checked the HTTP specification before open this issue : 
> Reason-Phrase  = *<TEXT, excluding CR, LF>
https://www.w3.org/Protocols/rfc2616/rfc2616-sec6.html

> *rule
> The character "*" preceding an element indicates repetition. The full form is "<n>*<m>element" indicating at least <n> and at most <m> occurrences of element. Default values are 0 and infinity so that "*(element)" allows any number, including zero; "1*element" requires at least one; and "1*2element" allows one or two.
https://www.w3.org/Protocols/rfc2616/rfc2616-sec2.html#sec2

Confirmed in this discussion : https://stackoverflow.com/questions/17517086/can-an-http-response-omit-the-reason-phrase

**Comment by urbanishimwe** [comment]
my bad! if this is the case `proto.HasResponseTitle` need modification too(it is used in Connection: Keep-Alive)

> but for my PR, I can't add too much change in your software

it will be fine as long as they are useful changes.

**Comment by davidFR** [comment]
Yes it is confirmed by #875 :  `proto.HasResponseTitle` failed for Response with Reason Phrase
